cluster-state-service/internal/features/wrappers/utils.go

```diff
@@ -40,3 +67,39 @@ func getECSEndpoint() (string, error) {
 
 	return endpoint, nil
 }
+
+func GetLatestECSOptimizedAMIID() (string, error) {
+	region := os.Getenv(regionEnvVarName)
+	if region == "" {
+		return "", fmt.Errorf("Empty region. Please specify the AWS region.")
+	}
+	return latestAMIIDByRegion[region], nil
```

Comment:
lets do the following to make sure there is nothing weird happening.

amiID, ok := latestAMIIDByRegion[region]
if !ok {
return "", errors.New(...)
}
return amiID, nil